packages/blockchain-wallet-v4/src/redux/data/coins/sagaRegister.ts

```diff
@@ -12,6 +12,7 @@ export default ({ api }) => {
     yield takeLatest(actions.fetchUnifiedBalances.type, dataCoinsSagas.fetchUnifiedBalances)
     yield takeLatest(actions.initializeSubscriptions.type, dataCoinsSagas.initializeSubscriptions)
     yield takeLatest(actions.unsubscribe.type, dataCoinsSagas.unsubscribe)
+    yield takeLatest(actions.subscribeToCoin.type, dataCoinsSagas.subscribe)
```

Comment:
nitpicky but can we match unsubscribe/subscribe naming?

either `unsubscribe` and `subscribe` or `unsubscribeFromCoin` and `subscribeToCoin`. I prefer the former